ftest/src/test/java/org/arquillian/reporter/GreeterVerifier.java

```diff
@@ -0,0 +1,160 @@
+package org.arquillian.reporter;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.List;
+import org.arquillian.core.reporter.impl.ArquillianCoreKey;
+import org.arquillian.environment.reporter.impl.EnvironmentKey;
+import org.arquillian.reporter.api.model.ReporterCoreKey;
+import org.arquillian.reporter.api.model.UnknownStringKey;
+import org.arquillian.reporter.api.model.report.ConfigurationReport;
+import org.arquillian.reporter.api.model.report.Report;
+import org.arquillian.reporter.api.model.report.TestClassReport;
+import org.arquillian.reporter.api.model.report.TestMethodReport;
+import org.arquillian.reporter.api.model.report.TestSuiteReport;
+import org.arquillian.reporter.impl.ExecutionReport;
+import org.arquillian.reporter.parser.ReportJsonParser;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.JUnitCore;
+
+import static org.arquillian.reporter.impl.asserts.ExecutionReportAssert.assertThatExecutionReport;
+import static org.arquillian.reporter.impl.asserts.ReportAssert.assertThatReport;
+import static org.arquillian.reporter.impl.asserts.TestClassReportAssert.assertThatTestClassReport;
+import static org.arquillian.reporter.impl.asserts.TestMethodReportAssert.assertThatTestMethodReport;
+import static org.arquillian.reporter.impl.asserts.TestSuiteReportAssert.assertThatTestSuiteReport;
+import static org.assertj.core.api.Assertions.assertThat;
+
+/**
+ * @author <a href="mailto:mjobanek@redhat.com">Matous Jobanek</a>
+ */
+public class GreeterVerifier {
```

Comment:
As we have discussed already, the environment report should be verified as well.